xclim/indices/_threshold.py

```diff
@@ -3423,15 +3427,26 @@ def wet_spell_total_length(
         Accumulated precipitation value over which a period is considered dry.
     window : int
         Number of days when the maximum or accumulated precipitation is over threshold.
-    op : {"max", "sum"}
+    op : {"min", "max", "sum"}
```

Comment:
I guess it should either allow "min" or "max" but not both, no?

Reply:
I mean, one could decide to define a "wet spell" with a max threshold, I'm not here to judge. I'll add the missing ones in the other functions.